DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -38,6 +42,37 @@ public EventsSheetsService()
                 HttpClientInitializer = credential,
                 ApplicationName = applicationName
             });
+
+            try
```

Comment:
I'd extract this to a method as the constructor is getting rather large 🙂

Reply:
Refactored in bf55cbc 🙂